src/main/java/com/adepuu/exercises/session7/Event.java

```diff
@@ -0,0 +1,196 @@
+package com.adepuu.exercises.session7;
+
+import java.util.*;
+
+public class Event {
+  private final List<Ticket> tickets = new ArrayList<>();
+  private final List<Ticket> bookedTickets = new ArrayList<>();
+  private final Scanner scanner = new Scanner(System.in);
+  User user = new User();
+
+  public void getTickets() {
+    try {
+      System.out.println("------List of Tickets-------");
+      if (tickets.isEmpty()) {
+        System.out.println("----- Tickets are empty -----");
+        return;
+      }
+      for (Ticket map : tickets) {
+        System.out.println("Ticket id: " + map.getTicketId());
+        System.out.println("Event name: " + map.getEventName());
+        System.out.println("Ticket name: " + map.getTicketName());
+        System.out.println("Price: " + map.getPrice());
+        System.out.println("--------------------------------");
+      }
+    } catch (Exception e) {
+      System.out.println("Ups Something Error");
+    }
+  }
+
+  public void create() {
+    System.out.println("""
+                        
+            ----- Create Ticket -----\s
+            """);
+
+    System.out.print("Event name: ");
+    String eventName = scanner.nextLine();
+
+
+    System.out.print("Ticket name: ");
+    String ticketName = scanner.nextLine();
+
+
+    System.out.print("Price: ");
+    String price = scanner.next();
+
+    try {
+      Ticket ticket = new Ticket(UUID.randomUUID().toString(), eventName, ticketName, price,"");
+      tickets.add(ticket);
+    } catch (Exception e) {
+      System.out.println("Ups Something Error");
+    }
+  }
+
+  public void editTicket() {
+    System.out.println("""
+                        
+            ----- Edit Ticket -----\s
+            """);
+    getTickets();
+
+    System.out.print("Insert Ticket id: ");
+    String ticketId = scanner.next();
+
+    System.out.print("Event name(Optional): ");
+    String eventName = scanner.next();
+
+    System.out.print("Ticket name(Optional): ");
+    String ticketName = scanner.next();
+
+    System.out.print("price(Optional): ");
+    String price = scanner.next();
+
+    if (Objects.equals(ticketId, "")) {
+      System.out.println("Ticket id cannot empty");
+      return;
+    }
+
+    try {
+      for (Ticket map : tickets) {
+        if (map.getTicketId().equals(ticketId)) {
+          map.setEventName(Objects.equals(eventName, "") ? map.getEventName() : eventName);
+          map.setTicketName(Objects.equals(ticketName, "") ? map.getTicketName() : ticketName);
+          map.setPrice(Objects.equals(price, "") ? map.getPrice() : price);
+        }
+      }
+      System.out.println("Edit ticket success");
+    } catch (Exception e) {
+      System.out.println("Ups something error");
+    }
```

Comment:
the same as above, the catch seems pointless since there's not enough info on what went wrong